StreamDebugger.h

```diff
@@ -9,43 +9,73 @@
 #ifndef StreamDebugger_h
 #define StreamDebugger_h
 
+#include <Arduino.h>
```

Comment:
This change is probably a breaking one: I see you support `energia` and `wiringpi`... But not having this include is really annoying (forces to have a correct include ordering).